crates/driver/openapi.yml

```diff
@@ -514,6 +522,37 @@ components:
           $ref: "#/components/schemas/TokenAmount"
         solver:
           $ref: "#/components/schemas/Address"
+    JitOrder:
+      type: object
+      properties:
+        sellToken:
+          $ref: "#/components/schemas/Address"
+        buyToken:
+          $ref: "#/components/schemas/Address"
+        sellAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        buyAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        receiver:
+          $ref: "#/components/schemas/Address"
+        validTo:
+          type: integer
+        side:
+          type: string
+          enum: ["buy", "sell"]
+        sellTokenSource:
+          type: string
+          enum: ["erc20", "internal", "external"]
+        buyTokenSource:
+          type: string
+          enum: ["erc20", "internal"]
+        appData:
+          type: string
+        signature:
+          type: string
```

Comment:
These signatures can be a bit finicky. Might be good to add a small explainer.
The not obvious details are:
* in case of `presign` this should contain the address of the owner
* in case of `eip1271` the signature should consist of `<owner_address><signature_bytes>` otherwise we'll not be able to recover the owner for this type of signature